CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 ### Fixes
 
 - Too long flush duration (#2370)
+- Do not delete the app state when OOM tracking is disabled (#2382)
```

Comment:
`m`: Please change this, so we describe what we are fixing instead of how. I think we fixed a bug that we always report cold starts when OOM is enabled.